### ISSUE TITLE
Make this work with Wordpress

### ISSUE THREAD

**Issue by kelvinramirez** [BlinkTagInc/gtfs-to-html#76]
Hello, Im working with a website using the plugin transit-custom-posts, I generated the timetables in HTML but I cant make it work in Wordpress, Im trying to understand the timetable.txt, also I have a bunch of timetables html, do I have to do the timetables hand made? Have you any of you guys make it work, can you help me with that?

**Comment by brendannee** [comment]
Were you able to upload the timetable HTML files to the directory specified in the "Manually uploading files" section of  https://trilliumtransit.github.io/transit-custom-posts/gtfs-update.html ?

**Comment by kelvinramirez** [comment]
Hi Team, 

Im able to make it work in Wordpress, but Im stock on some things the first one is: 

-Now Im able to get certain stops but im still having a lot of dots in the maps, how can I get only the dots corresponding to the stops I have in the table?
https://ripta.gladworksinprogress.com/ripta/20200101-20200622/15_71_1_mon-sat.html

The second:

-Im able to create the Mon-Friday, Saturday and Sundays using the timetables.txt and telling the timetable_page.txt that create a html with those three tables, I want this to create the tabs, im not sure this is the correct way to do this, the thing is that once I place this html generated I can not see the timetables in wordpress.
https://ripta.gladworksinprogress.com/ripta_other/20200101-20200622/route71.html

Basically we want to get this info:
https://www.ripta.com/71

This is my gtfsfiles:
https://ripta.gladworksinprogress.com/ripta_other/20200101-20200622/route71.html

Thank you very much.

**Comment by kelvinramirez** [comment]
I found out how to make the tabs, just adding "menuType": "radio" in config.json, But Im stucked in the other things.

**Comment by kelvinramirez** [comment]
I found how to add it in Wordpress already by naming the HTML pages like this id_route.html = 18_71.html, but Im still don´t know how to match the number of stops in the tables with the ones displayed in the maps.

**Comment by brendannee** [comment]
Right now, the map is set to show up all stops.
You could make your own version of the map javascript https://github.com/BlinkTagInc/gtfs-to-html/blob/master/public/js/timetable-map.js  which excludes only specific stops. Since "timepoint" is specified in stop_times.txt in GTFS not in stops, the GeoJSON generated for the map doesn't know which stops are timepoints and which are not.

This is a feature I could work on adding in the future.

**Comment by kelvinramirez** [comment]
Great I got you thank you very much Brendan, so the correct way to create the tabs is with timetables.txt and telling the timetable_page.txt that creates an html with those three tables?

like this:
timetable_page_id,timetable_page_label,filename
16,"Route 71 South Mon-Fri Saturday Sunday","route71.html"

Other thing can I use google maps?

**Comment by brendannee** [comment]
Yes - you can used timetable_pages.txt to create a page and then associate all three timetables  with that timetable page. 
Google maps can be used, but you'll need to rewrite  https://github.com/BlinkTagInc/gtfs-to-html/blob/master/public/js/timetable-map.js to use google maps  instead of mapbox

**Comment by kelvinramirez** [comment]
Can I decide between South or North to be display for default.
https://ripta.gladworksinprogress.com/ripta/20200101-20200622/18_71.html

Other thing can I ignore the rows without service?
https://ripta.gladworksinprogress.com/ripta/ignore_row.png

**Comment by brendannee** [comment]
Yes - you can specify the north timetable  to be first usiing
the timetable_sequence in timetables.txt which will cause it to be the
default.

There isn't an easy way to ignore specific trips - it looks like the trips
you highlighted in that image don't continue for the full route. Is there
actually a bus that runs for this trip, or is it some other issue?

**Comment by kelvinramirez** [comment]
Hi Brendan,

We are tying to replicate this table it look like its ignoring some trips, it looks like there is not buses at this time.
https://www.ripta.com/71

If we ignore the trips without services we can get the exact table
https://ripta.gladworksinprogress.com/ripta/20200101-20200622/18_71.html

Any Idea?

**Comment by brendannee** [comment]
You should remove the trips from the GTFS which are showing up that yo
udon't want. Each row has its  trip ID in the html id attribute so you can
easily see which trips are showing up that should be removed from the GTFS.

Alternatively, each row in the table has a unique ID - you could add CSS to
hide rows you do not want to show up. This could be a workaround if you
can't edit the GTFS.

**Comment by brendannee** [comment]
Marking this issue resolved - feel free to comment if you have more questions.
